src/shared/components/NotFound.tsx

```diff
@@ -154,7 +154,7 @@ const NotFound: FC = () => {
         org.current = await fetchOrg()
       }
     }
-    const deepLinkingMap = buildDeepLinkingMap(org.current)
+    const deepLinkingMap = buildDeepLinkingMap(org.current?.id)
```

Comment:
if this comes back as `undefined`, it will pass `undefined` into `buildDeepLinkingMap` which will cause a redirect to e.g. `/orgs/undefined/about` which will redirect to `/orgs`